avalon/tools/cbloader/app.py

```diff
@@ -171,7 +171,8 @@ def _versionschanged(self):
         if active:
             rows = selection.selectedRows(column=active.column())
             if active in rows:
-                node = active.data(subsets.model.NodeRole)
+                node_data = subsets.model.NodeRole
+                node = active.data(node_data)
```

Comment:
Personally I preferred the old style here: `node = active.data(subsets.model.NodeRole)`